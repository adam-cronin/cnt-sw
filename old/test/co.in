molecule{
0 1
C 0 0 0
O 1.15 0 0
}
set basis cc-pVDZ
energy('mp2')
