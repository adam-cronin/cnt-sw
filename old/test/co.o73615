hostname
node08.cluster
eval `/usr/bin/modulecmd tcsh load psi4conda/1.1`
/usr/bin/modulecmd tcsh load psi4conda/1.1
setenv LOADEDMODULES psi4conda/1.1
setenv PATH /usr/local/Dist/psi4conda/bin:/export/Linux-x86_64/bin:/export/Linux-x86_64/Dist/bin:/export/Linux-x86_64/sbin:/export/Linux-x86_64/global/bin:/export/Linux-x86_64/global/sbin:/home/acronin/bin:/bin:/sbin:/usr/bin:/usr/sbin:/usr/X11R6/bin:/usr/bin/X11:/export/Linux-x86_64/Dist/bin:/export/Linux-x86_64/sbin:/export/Linux-x86_64/global/bin:/export/Linux-x86_64/Dist/chimera/bin:/export/Linux-x86_64/Dist/Jmol/Jmol
setenv _LMFILES_ /usr/local/Modules/modulefiles/psi4conda/1.1
set _exit=0
/usr/bin/test 0 = 0
setenv PSI_SCRATCH /scratch/calpoly/acronin
psi4 -i co.in -o co.out -n 1
/usr/local/Dist/psi4conda/lib/python2.7/site-packages/v2rdm_casscf/v2rdm_casscf.so loaded.
