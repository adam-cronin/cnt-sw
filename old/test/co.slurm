#!/bin/tcsh
#SBATCH -p stdmem
#SBATCH --nodes=1
#SBATCH --ntasks-per-node=1
#SBATCH --mem=1GB
#SBATCH --export=ALL
#SBATCH -t 150:00:00
#SBATCH -J co
#SBATCH -o co.o%j

source /usr/share/Modules/init/tcsh
set echo
hostname

module load psi4conda/1.1
setenv PSI_SCRATCH /scratch/$GROUP/$USER
psi4 -i co.in -o co.out -n 1
