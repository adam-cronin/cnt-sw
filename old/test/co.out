
    -----------------------------------------------------------------------
          Psi4: An Open-Source Ab Initio Electronic Structure Package
                               Psi4 1.1 release

                         Git: Rev {HEAD} add49b9 


    R. M. Parrish, L. A. Burns, D. G. A. Smith, A. C. Simmonett,
    A. E. DePrince III, E. G. Hohenstein, U. Bozkaya, A. Yu. Sokolov,
    R. Di Remigio, R. M. Richard, J. F. Gonthier, A. M. James,
    H. R. McAlexander, A. Kumar, M. Saitow, X. Wang, B. P. Pritchard,
    P. Verma, H. F. Schaefer III, K. Patkowski, R. A. King, E. F. Valeev,
    F. A. Evangelista, J. M. Turney, T. D. Crawford, and C. D. Sherrill,
    J. Chem. Theory Comput. in press (2017).
    (doi: 10.1021/acs.jctc.7b00174)

    -----------------------------------------------------------------------


    Psi4 started on: Thursday, 05 November 2020 02:55AM

    Process ID:  98435
    PSIDATADIR: /usr/local/Dist/psi4conda/share/psi4
    Memory:     500.0 MiB
    Threads:    1
    
  ==> Input File <==

--------------------------------------------------------------------------
molecule{
0 1
C 0 0 0
O 1.15 0 0
}
set basis cc-pVDZ
energy('mp2')
--------------------------------------------------------------------------
    SCF Algorithm Type (re)set to DF.

*** tstart() called on node08.cluster
*** at Thu Nov  5 02:55:46 2020

   => Loading Basis Set <=

    Name: CC-PVDZ
    Role: ORBITAL
    Keyword: BASIS
    atoms 1 entry C          line   130 file /usr/local/Dist/psi4conda/share/psi4/basis/cc-pvdz.gbs 
    atoms 2 entry O          line   190 file /usr/local/Dist/psi4conda/share/psi4/basis/cc-pvdz.gbs 


         ---------------------------------------------------------
                                   SCF
            by Justin Turney, Rob Parrish, and Andy Simmonett
                              RHF Reference
                        1 Threads,    500 MiB Core
         ---------------------------------------------------------

  ==> Geometry <==

    Molecular point group: c2v
    Full point group: C_inf_v

    Geometry (in Angstrom), charge = 0, multiplicity = 1:

       Center              X                  Y                   Z               Mass       
    ------------   -----------------  -----------------  -----------------  -----------------
           C          0.000000000000     0.000000000000    -0.657053327808    12.000000000000
           O          0.000000000000     0.000000000000     0.492946672192    15.994914619560

  Running in c2v symmetry.

  Rotational constants: A = ************  B =      1.85916  C =      1.85916 [cm^-1]
  Rotational constants: A = ************  B =  55736.21726  C =  55736.21726 [MHz]
  Nuclear repulsion =   22.087396532452175

  Charge       = 0
  Multiplicity = 1
  Electrons    = 14
  Nalpha       = 7
  Nbeta        = 7

  ==> Algorithm <==

  SCF Algorithm Type is DF.
  DIIS enabled.
  MOM disabled.
  Fractional occupation disabled.
  Guess Type is SAD.
  Energy threshold   = 1.00e-08
  Density threshold  = 1.00e-08
  Integral threshold = 0.00e+00

  ==> Primary Basis <==

  Basis Set: CC-PVDZ
    Blend: CC-PVDZ
    Number of shells: 12
    Number of basis function: 28
    Number of Cartesian functions: 30
    Spherical Harmonics?: true
    Max angular momentum: 2

   => Loading Basis Set <=

    Name: (CC-PVDZ AUX)
    Role: JKFIT
    Keyword: DF_BASIS_SCF
    atoms 1 entry C          line   120 file /usr/local/Dist/psi4conda/share/psi4/basis/cc-pvdz-jkfit.gbs 
    atoms 2 entry O          line   220 file /usr/local/Dist/psi4conda/share/psi4/basis/cc-pvdz-jkfit.gbs 

  ==> Pre-Iterations <==

   -------------------------------------------------------
    Irrep   Nso     Nmo     Nalpha   Nbeta   Ndocc  Nsocc
   -------------------------------------------------------
     A1        14      14       0       0       0       0
     A2         2       2       0       0       0       0
     B1         6       6       0       0       0       0
     B2         6       6       0       0       0       0
   -------------------------------------------------------
    Total      28      28       7       7       7       0
   -------------------------------------------------------

  ==> Integral Setup <==

  ==> DFJK: Density-Fitted J/K Matrices <==

    J tasked:                  Yes
    K tasked:                  Yes
    wK tasked:                  No
    OpenMP threads:              1
    Integrals threads:           1
    Memory (MB):               375
    Algorithm:                Core
    Integral Cache:           NONE
    Schwarz Cutoff:          1E-12
    Fitting Condition:       1E-12

   => Auxiliary Basis Set <=

  Basis Set: (CC-PVDZ AUX)
    Blend: CC-PVDZ-JKFIT
    Number of shells: 48
    Number of basis function: 140
    Number of Cartesian functions: 162
    Spherical Harmonics?: true
    Max angular momentum: 3

  Minimum eigenvalue in the overlap matrix is 2.2330191149E-02.
  Using Symmetric Orthogonalization.

  SCF Guess: Superposition of Atomic Densities via on-the-fly atomic UHF.

  ==> Iterations <==

                           Total Energy        Delta E     RMS |[F,P]|

   @DF-RHF iter   0:  -113.02959665497673   -1.13030e+02   1.39138e-01 
   @DF-RHF iter   1:  -112.70170878599146    3.27888e-01   1.28874e-02 
   @DF-RHF iter   2:  -112.73171592385383   -3.00071e-02   7.59251e-03 DIIS
   @DF-RHF iter   3:  -112.74015024790994   -8.43432e-03   3.20579e-03 DIIS
   @DF-RHF iter   4:  -112.74397010199603   -3.81985e-03   1.62015e-03 DIIS
   @DF-RHF iter   5:  -112.74577274323846   -1.80264e-03   4.67927e-04 DIIS
   @DF-RHF iter   6:  -112.74592193460431   -1.49191e-04   1.21300e-04 DIIS
   @DF-RHF iter   7:  -112.74592434940240   -2.41480e-06   1.98720e-05 DIIS
   @DF-RHF iter   8:  -112.74592445297944   -1.03577e-07   3.63210e-06 DIIS
   @DF-RHF iter   9:  -112.74592445681624   -3.83680e-09   5.90172e-07 DIIS
   @DF-RHF iter  10:  -112.74592445686368   -4.74358e-11   3.24961e-08 DIIS
   @DF-RHF iter  11:  -112.74592445686392   -2.41585e-13   4.01619e-09 DIIS

  ==> Post-Iterations <==

    Orbital Energies (a.u.)
    -----------------------

    Doubly Occupied:                                                      

       1A1   -20.669828     2A1   -11.375692     3A1    -1.507937  
       4A1    -0.800098     1B2    -0.625057     1B1    -0.625057  
       5A1    -0.551343  

    Virtual:                                                              

       2B1     0.145641     2B2     0.145641     6A1     0.380967  
       3B1     0.671555     3B2     0.671555     7A1     0.724275  
       8A1     0.987340     9A1     1.060339     4B2     1.228284  
       4B1     1.228284    10A1     1.344660     1A2     1.344660  
       5B1     1.541788     5B2     1.541788    11A1     1.938768  
      12A1     2.299003    13A1     2.934881     2A2     2.934881  
       6B1     3.405608     6B2     3.405608    14A1     3.840566  

    Final Occupation by Irrep:
             A1    A2    B1    B2 
    DOCC [     5,    0,    1,    1 ]

  Energy converged.

  @DF-RHF Final Energy:  -112.74592445686392

   => Energetics <=

    Nuclear Repulsion Energy =             22.0873965324521748
    One-Electron Energy =                -197.3566186206244311
    Two-Electron Energy =                  62.5232976313083384
    DFT Exchange-Correlation Energy =       0.0000000000000000
    Empirical Dispersion Energy =           0.0000000000000000
    PCM Polarization Energy =               0.0000000000000000
    EFP Energy =                            0.0000000000000000
    Total Energy =                       -112.7459244568639178



Properties will be evaluated at   0.000000,   0.000000,   0.000000 Bohr

Properties computed using the SCF density matrix

  Nuclear Dipole Moment: (a.u.)
     X:     0.0000      Y:     0.0000      Z:     0.0024

  Electronic Dipole Moment: (a.u.)
     X:     0.0000      Y:     0.0000      Z:    -0.1369

  Dipole Moment: (a.u.)
     X:     0.0000      Y:     0.0000      Z:    -0.1345     Total:     0.1345

  Dipole Moment: (Debye)
     X:     0.0000      Y:     0.0000      Z:    -0.3418     Total:     0.3418


*** tstop() called on node08.cluster at Thu Nov  5 02:55:47 2020
Module time:
	user time   =       0.36 seconds =       0.01 minutes
	system time =       0.03 seconds =       0.00 minutes
	total time  =          1 seconds =       0.02 minutes
Total time:
	user time   =       0.36 seconds =       0.01 minutes
	system time =       0.03 seconds =       0.00 minutes
	total time  =          1 seconds =       0.02 minutes

*** tstart() called on node08.cluster
*** at Thu Nov  5 02:55:47 2020


  //>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>//
  //               DFMP2               //
  //<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<//

   => Loading Basis Set <=

    Name: (CC-PVDZ AUX)
    Role: RIFIT
    Keyword: DF_BASIS_MP2
    atoms 1 entry C          line   158 file /usr/local/Dist/psi4conda/share/psi4/basis/cc-pvdz-ri.gbs 
    atoms 2 entry O          line   234 file /usr/local/Dist/psi4conda/share/psi4/basis/cc-pvdz-ri.gbs 

	 --------------------------------------------------------
	                          DF-MP2                         
	      2nd-Order Density-Fitted Moller-Plesset Theory     
	              RMP2 Wavefunction,   1 Threads             
	                                                         
	        Rob Parrish, Justin Turney, Andy Simmonett,      
	           Ed Hohenstein, and C. David Sherrill          
	 --------------------------------------------------------

   => Auxiliary Basis Set <=

  Basis Set: (CC-PVDZ AUX)
    Blend: CC-PVDZ-RI
    Number of shells: 36
    Number of basis function: 112
    Number of Cartesian functions: 132
    Spherical Harmonics?: true
    Max angular momentum: 3

	 --------------------------------------------------------
	                 NBF =    28, NAUX =   112
	 --------------------------------------------------------
	   CLASS    FOCC     OCC    AOCC    AVIR     VIR    FVIR
	   PAIRS       0       7       7      21      21       0
	 --------------------------------------------------------

	-----------------------------------------------------------
	 ==================> DF-MP2 Energies <==================== 
	-----------------------------------------------------------
	 Reference Energy          =    -112.7459244568639178 [Eh]
	 Singles Energy            =      -0.0000000000000004 [Eh]
	 Same-Spin Energy          =      -0.0784848418448653 [Eh]
	 Opposite-Spin Energy      =      -0.2162495103136558 [Eh]
	 Correlation Energy        =      -0.2947343521585216 [Eh]
	 Total Energy              =    -113.0406588090224460 [Eh]
	-----------------------------------------------------------
	 ================> DF-SCS-MP2 Energies <================== 
	-----------------------------------------------------------
	 SCS Same-Spin Scale       =       0.3333333333333333 [-]
	 SCS Opposite-Spin Scale   =       1.2000000000000000 [-]
	 SCS Same-Spin Energy      =      -0.0261616139482884 [Eh]
	 SCS Opposite-Spin Energy  =      -0.2594994123763870 [Eh]
	 SCS Correlation Energy    =      -0.2856610263246758 [Eh]
	 SCS Total Energy          =    -113.0315854831885929 [Eh]
	-----------------------------------------------------------


*** tstop() called on node08.cluster at Thu Nov  5 02:55:47 2020
Module time:
	user time   =       0.08 seconds =       0.00 minutes
	system time =       0.00 seconds =       0.00 minutes
	total time  =          0 seconds =       0.00 minutes
Total time:
	user time   =       0.44 seconds =       0.01 minutes
	system time =       0.03 seconds =       0.00 minutes
	total time  =          1 seconds =       0.02 minutes

*** Psi4 exiting successfully. Buy a developer a beer!
